criu/cr-restore.c

```diff
@@ -3240,7 +3240,7 @@ static bool groups_match(gid_t* groups, int n_groups)
 	n = getgroups(0, NULL);
 	if (n == -1) {
 		pr_perror("Failed to get number of supplementary groups");
-		ret = false;
+		return false;
```

Comment:
should we return an error in this case?

Reply:
Good question. I do not know. If we already print out errno, so it seems it is an error. The man-page says:
`On error, -1 is returned, and errno is set appropriately.`. This was just recently added:
```
commit 0d6b8be72521232261ec1872522fb6977e0a4085
Author: Angie Ni <avtni@google.com>
Date:   Fri Jun 12 10:20:28 2020 -0600

    restore: skip unnecessary setgroups calls
    
    When groups already have desired values, we skip calling setgroups.
    Furthermore, this check allows us to restore when in an
    unprivileged user namespace that cannot use setgroups.
```